### PR TITLE
first map render

### PR DESCRIPTION
**Pull request by giuliamml**
First draft of the map component.
Rendered the map on the homepage using Mapbox API.

To do:
- [x] add access token
- [x] add marker (first iteration)
Screenshot 2021-03-20 at 13 04 29

### CODE REVIEW

File: src/pages/home.tsx [hunk 1]
```diff
@@ -38,9 +40,7 @@ class Home extends Component<{}, IHomeState> {
     );
```
Comment (pim0223) [inline src/pages/home.tsx:40]: You can remove the old `chainsMarkup`, as it is no longer used

File: src/util/map.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
+import React, { Component } from "react";
+import mapboxgl from "mapbox-gl";
+
+mapboxgl.accessToken =
+"pk.eyJ1IjoiZ2l1bGlhbW1sIiwiYSI6ImNrbWRiZjM2cDI3eXoyem16MXRtZGZ3ajMifQ.KVY1mrgZdLwLzqcIsYOQrg";
```
Comment (pim0223) [inline src/util/map.js:5]: Probably best to store this in an environment variable, have you worked with those?

**Comment by pim0223** [comment]
2 small comments, looks good!

**Comment by pim0223** [comment]
1 more thing: I would maybe put `Map` in `components/`, what do you think?

File: src/util/map.js (hunk 2 above)
Comment (timstokman) [reply src/util/map.js:5]: Probably should start rotating this token, it's already leaked

File: package-lock.json [hunk 3]
```diff
@@ -0,0 +1,18993 @@
+{
```
Comment (timstokman) [inline package-lock.json:1]: Use yarn instead of npm please

File: src/util/map.js (hunk 2 above)
Comment (giuliamml) [reply src/util/map.js:5]: yeah that makes total sense. Passing the token as an environment variable and storing it in a .env file. That's the code:

`map.js`:
```mapboxgl.accessToken = 
`${process.env.REACT_APP_MAPBOX_KEY}`;
```

and the .env file:

```
REACT_APP_MAPBOX_KEY = ...
```

File: package-lock.json (hunk 3 above)
Comment (giuliamml) [reply package-lock.json:1]: yes, will amend this in the next commit

File: yarn 2.lock [hunk 4]
```diff
@@ -0,0 +1,12469 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (timstokman) [inline yarn 2.lock:1]: yarn 2.lock? What is up with this?

Comment (giuliamml) [reply yarn 2.lock:1]: Might have been added when I've reinstalled yarn (possibly?!). Shall I remove it?

Comment (timstokman) [reply yarn 2.lock:1]: Yes please. We already have a lock file.

**Review by pim0223 — APPROVED** [review]

**Pull request merged**